lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/dataaccess/AccessApprovalStatusResult.json

```diff
@@ -0,0 +1,26 @@
+{
+	"description": "Result of a single access approval status request.",
```

Comment:
change this DTO according to https://sagebionetworks.jira.com/wiki/display/PLFM/Solution+and+Services